rustfmt-core/rustfmt-lib/tests/parser/issue-4126/invalid.rs

```diff
@@ -0,0 +1,6 @@
+fn foo() {
```

Comment:
I elected to stash the test files in a new directory under `tests` (`parser`) since they're explicitly testing for fatal errors during formatting as opposed to our typical integration/idempotence tests.

I'm open to another directory name and/or location, but I do think it's important to have negative scenario tests like this to help give us better protection against upstream changes in rustc parsing and prevent any regressions.